### ISSUE TITLE
Manipulate XML-Tags

### ISSUE THREAD

**Issue by AndreasDrescher** [wojtpl2/ExtendedXmlSerializer#4]
Hey Wojtpl2,

very nice Feature!!!! Thx, but how can i manipulate the generated XML-Tags and XML-Attribute?

Example: 

```
<?xml version="1.0" encoding="utf-8"?>
<Definitions type="BPMN2.Definitions">
  <id>text</id>
<Exports>text</Exports>
</Definitions>
```

i want to generate this

```
<?xml version="1.0" encoding="utf-8"?>
<Definitions id="text">
<Exports>text</Exports>
</Definitions>
```

with e.g. [XmlElement("Exports")] or  [XmlAttribute("id")] 

Can u help me? THX!

**Comment by wojtpl2** [comment]
You're right. You can make pull request or you have to wait about 1 week. If you would like to make pull request I can give some advice where and what to do.

**Comment by AndreasDrescher** [comment]
Thanks, but my programming skills are not excellent, so I think I'd rather wait a week;).

**Comment by wojtpl2** [comment]
Sorry for the delay.
I've published version 1.4.0. I support XmlElement and XmlRoot atributes. For XmlAttribute i need more time. Please check if this solves your problem.

**Comment by wojtpl2** [comment]
Adding support for XmlAttribute was moved to #6
